### ISSUE TITLE
`prefer-early-return`/`prefer-continue`: Add option for enforcing no early return/no continue when less than `maximumStatements`

### ISSUE THREAD

**Issue by abrahamguo** [sindresorhus/eslint-plugin-unicorn#3691]
### Description

The two rules' default of `maximumStatements: 1` expresses the philosophy that the following is allowed:

```js
if (condtion) return 5;
```

Since that is allowed, I propose that the following should be flagged and rewritten, since it's a less clear, and more wordy, way of writing the same:

```js
if (!condition) return;
return 5;
```

### Examples

```js
// ❌
function foo() {
	if (!condition) return;
	doSomething();
}

// ✅
function foo() {
	if (condition) doSomething();
}

```

```js
// ❌
for (const item of items) {
	if (!item.isActive) continue;
	else {
		process(item);
	}
}

// ✅
for (const item of items) {
	if (item.isActive) {
		process(item);
	}
}
```
